### PR TITLE
Handle headers

### PR DESCRIPTION
**Pull request by kalys**

### CODE REVIEW

File: lib/bamboo/adapters/ses_adapter.ex [hunk 1]
```diff
@@ -40,6 +40,16 @@ defmodule Bamboo.SesAdapter do
     end
   end
 
+  def put_headers(message, headers) when is_map(headers) do
```
Comment (AdiletAbylov) [inline lib/bamboo/adapters/ses_adapter.ex:43]: if there is only one line of code inside function use short form

File: test/lib/bamboo/adapters/ses_adapter_test.exs [hunk 2]
```diff
@@ -109,6 +109,24 @@ defmodule Bamboo.SesAdapterTest do
...
+    expected_request_fn = fn _, _, body, _, _ ->
+      message = parse_body(body)
+
+      assert Mail.Message.get_header(message, :x_custom_header) == "header-value; another-value"
+      assert Mail.Message.get_header(message, :reply_to) == "chuck@example.com"
+
+      {:ok, %{status_code: 200}}
+    end
+
+    HttpMock
```
Comment (AdiletAbylov) [inline test/lib/bamboo/adapters/ses_adapter_test.exs:122]: If there is only one function call in pipe, don't use pipe

File: lib/bamboo/adapters/ses_adapter.ex (hunk 1 above)
Comment (AdiletAbylov) [inline lib/bamboo/adapters/ses_adapter.ex:43]: Where is docs?
Where is specs?

Comment (kalys) [reply lib/bamboo/adapters/ses_adapter.ex]: I made those methods private.

**Pull request merged**